### PR TITLE
P1 rating

### PR DESCRIPTION
**Pull request by SJTGSHIVAM**
Added Rating component
live link for deploy https://deploy-preview-7--thisui.netlify.app/molecules/rating

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const drawerData = [
   { innerText: "Image", href: "/atoms/image" },
   { innerText: "Input", href: "/atoms/input" },
   { innerText: "Grid", href: "/atoms/grid" },
-  // { innerText: "Dummy", href: "/atoms/typography" },
+  { innerText: "Navbar", href: "/molecules/navbar" },
   // { innerText: "Dummy", href: "/atoms/typography" },
```
Comment (SJTGSHIVAM) [inline index.js:20]: need to add rating as well

File: assets/svgs/cart.svg [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+	<circle cx="28.25" cy="75.8" r="6.5" style="stroke: none; stroke-width: 1; stroke-dasharray: none; stroke-linecap: butt; stroke-linejoin: miter; stroke-miterlimit: 10; fill: rgb(0,0,0); fill-rule: nonzero; opacity: 1;" transform="  matrix(1 0 0 1 0 0) "/>
+	<circle cx="68.28999999999999" cy="75.8" r="6.5" style="stroke: none; stroke-width: 1; stroke-dasharray: none; stroke-linecap: butt; stroke-linejoin: miter; stroke-miterlimit: 10; fill: rgb(0,0,0); fill-rule: nonzero; opacity: 1;" transform="  matrix(1 0 0 1 0 0) "/>
+	<path d="M 19.306 23.417 c -1.374 0 -2.613 -0.95 -2.925 -2.347 l -1.375 -6.155 c -0.554 -2.48 -2.716 -4.212 -5.258 -4.212 H 3 c -1.657 0 -3 -1.343 -3 -3 s 1.343 -3 3 -3 h 6.749 c 5.372 0 9.942 3.662 11.113 8.904 l 1.375 6.155 c 0.361 1.617 -0.657 3.221 -2.274 3.582 C 19.742 23.393 19.522 23.417 19.306 23.417 z" style="stroke: none; stroke-width: 1; stroke-dasharray: none; stroke-linecap: butt; stroke-linejoin: miter; stroke-miterlimit: 10; fill: rgb(0,0,0); fill-rule: nonzero; opacity: 1;" transform=" matrix(1 0 0 1 0 0) " stroke-linecap="round" />
+</g>
+</g>
+</svg>
```
Comment (varnita619) [inline assets/svgs/cart.svg]: You can add a new line at the end of each file
you can also enable addFinalNewLine in vscode settings

**Review by mdarif6 — APPROVED** [review]
looking good

**Review by varnita619 — APPROVED** [review]
Rest all looks good and clean

**Review by Hnh013 — APPROVED** [review]
rating is working accordingly in example as well , code is good to approve and merge

**Review by samyakshah3008 — APPROVED** [review]
Loved the rating feature
Looks good to merge!

**Pull request merged**